packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -2774,6 +2812,50 @@ With a value of:
 "
 `;
 
+exports[`.toHaveProperty() {pass: true} expect({"a": {"b": [1, 2, 3]}}).toHaveProperty('a,b,1')' 1`] = `
+"<dim>expect(</><red>object</><dim>).not.toHaveProperty(</><green>path</><dim>)</>
+
+Expected the object:
+  <red>{\\"a\\": {\\"b\\": [1, 2, 3]}}</>
+Not to have a nested property:
+  <green>[\\"a\\", \\"b\\", 1]</>
+"
+`;
+
+exports[`.toHaveProperty() {pass: true} expect({"a": {"b": [1, 2, 3]}}).toHaveProperty('a,b,1', 2) 1`] = `
+"<dim>expect(</><red>object</><dim>).not.toHaveProperty(</><green>path</>, <green>value</><dim>)</>
+
+Expected the object:
+  <red>{\\"a\\": {\\"b\\": [1, 2, 3]}}</>
+Not to have a nested property:
+  <green>[\\"a\\", \\"b\\", 1]</>
+With a value of:
+  <green>2</>
+"
+`;
+
+exports[`.toHaveProperty() {pass: true} expect({"a": {"b": {"c": {"d": 1}}}}).toHaveProperty('a,b,c,d')' 1`] = `
```

Comment:
Btw, this is out of scope of this PR, but could you please remove this extra single-quote [here](https://github.com/facebook/jest/pull/5220/files#diff-a543b9f996079746ea774e265accc10fR806)? 
Also, not super relevant, but this should say `.toHaveProperty(['a','b','c','d'])` instead `.toHaveProperty('a,b,c,d')`

Reply:
I fixed the changelog and remove the extra single-quote.  How would you prefer for the snapshot output be changed (if at all for this PR)?  I tried using the `stringify` util function but ended up with a different issue where the output looked like `.toHaveProperty('["a","b","c","d"]')`

Reply:
If I had to choose, I'd pick the stringified version (it's just a name of the test anyway)

Reply:
Taking a second look, it also had an adverse effect of adding addition double quotes to normal keypaths like so : `exports[`.toHaveProperty() {pass: false} expect("abc").toHaveProperty('"a.b.c"') 1`]`.  This might be better off fixed in the `pretty-format` package to prevent really cluttering up the matcher tests with string parsing logic.